From: Hideki Yamane <henrich@debian.org>
Date: Sat, 19 Nov 2016 21:25:32 +0900
Subject: manpages should be in /usr/share/man

Filesystem Hierarchy Standard (on Linux), at least.
And FreeBSD, see https://www.freebsd.org/cgi/man.cgi?query=filesystem&manpath=SunOS+4.1.3
---
 Makefile | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -50,14 +50,14 @@
 	cp -f nkf $(DESTDIR)/$(prefix)/bin/
 
 install-man:
-	-$(MKDIR) -p $(DESTDIR)/$(prefix)/man
-	-$(MKDIR) -p $(DESTDIR)/$(prefix)/man/man1
-	cp -f nkf.1 $(DESTDIR)/$(prefix)/man/man1/
+	-$(MKDIR) -p $(DESTDIR)/$(prefix)/share/man
+	-$(MKDIR) -p $(DESTDIR)/$(prefix)/share/man/man1
+	cp -f nkf.1 $(DESTDIR)/$(prefix)/share/man/man1/
 
 install-man-ja:
-	-$(MKDIR) -p $(DESTDIR)/$(prefix)/man/ja
-	-$(MKDIR) -p $(DESTDIR)/$(prefix)/man/ja/man1
-	cp -f nkf.1j $(DESTDIR)/$(prefix)/man/ja/man1/nkf.1
+	-$(MKDIR) -p $(DESTDIR)/$(prefix)/share/man/ja
+	-$(MKDIR) -p $(DESTDIR)/$(prefix)/share/man/ja/man1
+	cp -f nkf.1j $(DESTDIR)/$(prefix)/share/man/ja/man1/nkf.1
 
 shar:
 	-mkdir nkf-$(VERSION)
